query/src/sessions/query_ctx.rs

```diff
@@ -394,7 +390,7 @@ impl QueryContextShared {
     pub(in crate::sessions) fn destroy_context_ref(&self) {
         if self.ref_count.fetch_sub(1, Ordering::Release) == 1 {
             std::sync::atomic::fence(Acquire);
-            tracing::debug!("Destroy QueryContext");
+            tracing::warn!("Destroy QueryContext");
```

Comment:
is it a temporary "upgrade" of log level? for the testing of this PR only? : )

Reply:
Let's me roll it back.